src/platform/static-data/footer-links.json

```diff
@@ -0,0 +1,310 @@
+[
+  {
+    "column": "1",
+    "href": "https://www.va.gov/homeless/",
+    "order": "1",
+    "target": "",
+    "title": "For Homeless Veterans"
+  },
+  {
+    "column": "1",
+    "href": "https://www.va.gov/womenvet/",
+    "order": "2",
+    "target": "",
+    "title": "For Women Veterans"
+  },
+  {
+    "column": "1",
+    "href": "https://www.va.gov/centerforminorityveterans/",
+    "order": "3",
+    "target": "",
+    "title": "For Minority Veterans"
+  },
+  {
+    "column": "1",
+    "href": "https://www.ptsd.va.gov",
+    "order": "4",
+    "target": "",
+    "title": "PTSD"
+  },
+  {
+    "column": "1",
+    "href": "https://www.mentalhealth.va.gov",
+    "order": "5",
+    "target": "",
+    "title": "Mental Health"
+  },
+  {
+    "column": "1",
+    "href": "https://www.va.gov/adaptivesports/",
+    "order": "6",
+    "target": "",
+    "title": "Adaptive Sports and Special Events"
```

Comment:
Instead of the column property, would it possibly be easier to use a structure with the column values as keys?

```
{
    "column_1": // array of values that previously had "1" as their "column" value
}
```

Reply:
I'd rather keep it as an array right now, seems simpler if content people need to change or update a column. Happy to update if you think it works better as an obj though.